### PR TITLE
Rancher integration updates

### PR DESCRIPTION
**Pull request by noahgildersleeve**
These are the tests that came out of the talks about Harvester Rancher integration. Also standardized test cases on non-spcific ordered lists in markdown.

### CODE REVIEW

**Review by TachunLin — COMMENTED** [review]
Test case `49-overprovision-harvester` file name mismatch with title `48-Verify Backup and restore on server migration`. This may cause duplicate test case name display on github page

Beside this, other cases looks good to me.
 
image

File: docs/content/manual/harvester-rancher/49-overprovision-harvester.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: 48-Verify Backup and restore on server migration
```
Comment (lanfon72) [inline docs/content/manual/harvester-rancher/49-overprovision-harvester.md:2]: title should be updated.

File: docs/content/manual/harvester-rancher/41-rancher-nested-harvester [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
```
Comment (lanfon72) [inline docs/content/manual/harvester-rancher/41-rancher-nested-harvester:1]: file named without suffix `.md`

**Review by lanfon72 — CHANGES_REQUESTED** [review]

File: docs/content/manual/harvester-rancher/49-overprovision-harvester.md (hunk 1 above)
Comment (noahgildersleeve) [reply docs/content/manual/harvester-rancher/49-overprovision-harvester.md]: Changed.

File: docs/content/manual/harvester-rancher/41-rancher-nested-harvester (hunk 2 above)
Comment (noahgildersleeve) [reply docs/content/manual/harvester-rancher/41-rancher-nested-harvester]: Changed.

**Comment by lanfon72** [comment]
LGTM

**Review by lanfon72 — APPROVED** [review]

**Pull request merged**